packages/modeldb-sqlite-wasm/src/TransientModelDB.ts

```diff
@@ -0,0 +1,73 @@
+import { logger } from "@libp2p/logger"
+import sqlite3InitModule from "@sqlite.org/sqlite-wasm"
+import {
+	AbstractModelDB,
+	parseConfig,
+	Effect,
+	ModelValue,
+	ModelSchema,
+	QueryParams,
+	Config,
+	getFilter,
+} from "@canvas-js/modeldb"
+import { InnerModelDB } from "./InnerModelDB.js"
+import { Awaitable } from "@canvas-js/interfaces"
+import assert from "assert"
```

Comment:
`assert` is a NodeJS package and we don't want to make people polyfill things if necessary. we export our own as `import { assert } from "@canvas-js/utils"`

Reply:
Good catch